DirectProgramming/DPC++/N-BodyMethods/Nbody/src/GSimulation.cpp

```diff
@@ -0,0 +1,230 @@
+//==============================================================
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+// =============================================================
+// Copyright (c) 2019 Fabio Baruffa
+// Source: https://github.com/fbaru-dev/particle-sim
+// MIT License
+// =============================================================
+
+#include "GSimulation.hpp"
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities/latest/include/dpc_common.hpp
+#include "dpc_common.hpp"
+using namespace sycl;
```

Comment:
A blank line here as well

Reply:
This change is already done and pushed in.